src/realm.js

```diff
@@ -885,6 +887,7 @@ export class Realm {
         effects1 = Widen.widenEffects(this, effects1, effects2);
       }
     } catch (e) {
+      if (e instanceof InvariantError) throw e;
```

Comment:
Why would it ever be okay to swallow random errors? What about SyntaxError or RangeError or TypeError? Any error that we don't know if benign for sure is by default a bug.